etc/cnid_dbd/meson.build

```diff
@@ -1,73 +1,81 @@
-cnid_dbd_sources = [
-    'comm.c',
-    'db_param.c',
-    'dbd_add.c',
-    'dbd_dbcheck.c',
-    'dbd_delete.c',
-    'dbd_get.c',
-    'dbd_getstamp.c',
-    'dbd_lookup.c',
-    'dbd_rebuild_add.c',
-    'dbd_resolve.c',
-    'dbd_search.c',
-    'dbd_update.c',
-    'dbif.c',
-    'main.c',
-    'pack.c',
-]
+if use_dbd_backend
```

Comment:
Is there an argument to be made to always enable the dbd backend?

The argument to always enable: It is the one single stable and full-featured CNID backend presently.

The argument to make it optional: The potential of creating a new CNID backend in the future (e.g. sqlite) which will become the default.